javaslang/src/main/java/javaslang/control/Try.java

```diff
@@ -1198,9 +1198,11 @@ private WithResources1(CheckedSupplier<? extends T1> t1Supplier) {
         @SuppressWarnings("try")/* https://bugs.openjdk.java.net/browse/JDK-8155591 */
         public <R> Try<R> of(CheckedFunction1<? super T1, ? extends R> f) {
             return Try.of(() -> {
+                final R result;
                 try (T1 t1 = t1Supplier.get()) {
-                    return f.apply(t1);
+                    result = f.apply(t1);
```

Comment:
It seems you have to choose between @sputnikci and @codecov-io complaining :p
Could we open a @codecov-io issue instead of making the code uglier?